iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalCatalog.java

```diff
@@ -101,16 +96,16 @@ public boolean dropTable(TableIdentifier identifier, boolean purge) {
           houseTableRepositoryException);
     }
     if (purge) {
-      // delete data and metadata files on hdfs
-      try {
-        FileSystem fs = fsStorageProvider.storageClient();
-        fs.delete(new Path(tableLocation), true);
-      } catch (IOException e) {
-        throw new UncheckedIOException(
-            String.format(
-                "Deleting Directory Failed databaseId: %s, tableId: %s, tableLocation: %s",
-                identifier.namespace().toString(), identifier.name(), tableLocation),
-            e);
+      // Delete data and metadata files from storage.
+      if (fileIO instanceof SupportsPrefixOperations) {
```

Comment:
how can we test this ? cc @HotSushi

Reply:
Can we run a manual test on docker and ensure hdfs directory gets dropped post delete?

Reply:
Yes. I tested using docker. Also, the e2e tests for service invoke dropTable.